proposed/psr-8-hug/psr-8-hug.md

```diff
@@ -56,9 +64,21 @@ interface Huggable
 {
 
     /**
+     * Asks whether a hug will be reciprocated.
+     *
+     * Well-behaved Huggables MAY use this method to verify that their
+     * invocation of hug() will be handled by target object.
+     *
+     * @param Huggable $h
+     *   The object that desires a hug.
+     * @return boolean
+     */
+    public function willHug(Huggable $h);
```

Comment:
Is `willHug` the correct technical term? Or does `isWillingToHug` or `wouldHug` fit better?